analysisservice/src/main/java/nu/yona/server/AnalysisServiceApplication.java

```diff
@@ -51,6 +51,6 @@ public Docket yonaApi()
 	@Bean
 	public CacheManager cacheManager()
 	{
-		return new ConcurrentMapCacheManager("goalConflictMessages");
+		return new ConcurrentMapCacheManager("goalConflictMessages", "usersAnonymized");
```

Comment:
Accept. Will change this later. However for the app service you do not strictly need the `"goalConflictMessages"` cache.